app/models/spree/shipment.rb

```diff
@@ -103,12 +103,14 @@ def selected_shipping_rate_id=(id)
       save!
     end
 
-    def refresh_rates
+    def refresh_rates(checkout: true, apply_tags: true)
```

Comment:
one of the places where this is called without args is the order model on refresh_shipment_rates which is only used in specs, in the order factory, we can move this method to the factory.

the other place is the ShipmentsController which is used in the backoffice.

So, I think the correct default values here are false and false. This means that I think we dont need these args in this method. You can just call refresh_rates as before and not have these args. You can then call the estimator with false, false inside this method.

Reply:
to be more clear, the code would be:

```ruby
def refresh_rates
```

and below:
```ruby
      self.shipping_rates = OrderManagement::Stock::Estimator.new(order).shipping_rates(
        package: to_package, checkout: false, apply_tags: false
      )
```

Reply:
Good catch, I didn't see that that one call was only used in specs. 

da64081067547357683cc1453ad1e9be2c0302f7